hack/run_test

```diff
@@ -308,21 +295,29 @@ function run_replication_test() {
       sleep 1
     done
   done
+}
 
+function test_value_replication() {
+  local table_name="$1"; shift
+  local max_attempts=30
+
+  # Setup the replication data
   local value
   value=24
-  # Do some real work to test replication in practice
-  postgresql_cmd -c "CREATE TABLE t1 (a integer); INSERT INTO t1 VALUES ($value);"
+  local table_name="t1"
+  postgresql_cmd -c "CREATE TABLE $table_name (a integer); INSERT INTO $table_name VALUES ($value);"
 
   # Read value from slaves and check whether it is expected
   local f
   local slave_ip
+  local cidfile
+  local master_ip=$(get_container_ip master-$cid_suffix.cid)
   for f in `find ${CIDFILE_DIR} -type f -name "slave-*"`; do
```

Comment:
> We only need the IPs now, but we might need something else in the future.

The point was not to docker inspect all the time, but that is just preference.  Container's id is better than doing error-prone find command.

> We will still need unique file names for the cid files in order for the containers to be cleaned up properly, so the $suffix will probably need to stay in.

Right, the suffix is already there.  But this particular find could go away completely.